cupy/fft/config.py

```diff
@@ -1,3 +1,30 @@
-
-global enable_nd_planning
 enable_nd_planning = True
+use_multi_gpus = False
+_devices = None
+
+
+def set_cufft_gpus(gpus):
```

Comment:
We don't want this API because we want users to declare `Pland1d` explicitly in such a situation.

Reply:
May I ask what is the concern here? I thought supporting high-level APIs would be appreciated...

Reply:
And I was under the impression that you were on board, since you marked this https://github.com/cupy/cupy/pull/2644#discussion_r350168492 as resolved. To remove this, I will also have to undo all the docs I just wrote 🙁

Reply:
...and I just realized tests have to be rewritten too, as I didn't test `Plan1d` directly. 😞

Reply:
I did not doubt these API when I marked the comment as resolved, but now I think that high-level APIs are not necessary for low-level performance improvement. Sorry to ask you for redundant work.

Reply:
In this case, I can't reuse all of the existing FFT tests. Do you need a separate test for `Plan1d`?

Reply:
@asi1024 How about my above question for the tests? And do you want everything here to be removed (`use_multi_gpus`, `_devices`, and `set_cufft_gpus`)?

Reply:
Sorry, I thought we could get `device` from `plan`, but actually `Plan1d` does not have `device` attribute.

Reply:
However, this function may be removed in the future. CuPy does not support multiple-GPU computation in one operation currently, but it may be supported in the future, and a feature for multiple-GPU device management may also be supported.

Reply:
Thanks, @asi1024! So if I understand you correctly, it is fine to keep `set_cufft_gpus()` as is. I suppose I better mark it experimental?

Reply:
Yes!

Reply:
I already did it. Thanks.